frontend/src/components/App/App.js

```diff
@@ -64,22 +48,22 @@ function App() {
     // }, (dataBlockDuration / 60) * 2000);
   }
 
-  const fadeAllStations = () => {
-    allStations.forEach((line) => {
-      line.forEach((station) => {
-        // console.log(station);
-        document.getElementById(station
-          .replace(/ *\([^)]*\) */g, "")
-          .replace(/\s|\.''/g, '')
-          .replace(/\./g, '')
-          .replace(/'/g, '')
-          .replace(/UndergroundStation/g, '')
-          .replace(/-Underground/g, '')
-          .replace(/&/g, '_'),)
-        .style.opacity = "0%";
-      });
-    });
-  }
+  // const fadeAllStations = () => {
+  //   allStations.forEach((line) => {
+  //     line.forEach((station) => {
```

Comment:
Dont be trigger shy ... lets delete these dudes! Great idea to move them to their own file